packages/block-library/src/image/save.js

```diff
@@ -58,6 +58,8 @@ export default function save( { attributes } ) {
 				...borderProps.style,
 				aspectRatio,
 				objectFit: scale,
+				width,
+				height,
```

Comment:
These need units since only numbers are saved.

Reply:
Since they're only output inside the HTML attrs of IMG they are wroking as they're here. I think we need to come back to it with your ideas of allowing this other unites and make them strings.

Reply:
It seems it works without units by virtue of React being awesome. If the keys are width and height in a styles attribute in JSX the numbers will default to 'px`